experimental/linting_tool/lib/widgets/routes.dart

```diff
@@ -0,0 +1,5 @@
+// Copyright 2021 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const String homeRoute = '/home';
```

Comment:
Are we going to see more routes? Another TODO?

Reply:
Yup, adding.